### PR TITLE
Remove slack update during updating events

### PR DESCRIPTION
**Pull request by simon-kwame**

### CODE REVIEW

**Review by csalvato — APPROVED** [review]
If this solves our problem, LGTM!

File: lib/slackistrano/tasks/slack.rake [hunk 1]
```diff
@@ -42,6 +42,5 @@ end
 before 'deploy:starting',           'slack:deploy:starting'
 before 'deploy:updating',           'slack:deploy:updating'
 before 'deploy:reverting',          'slack:deploy:reverting'
-after  'deploy:finishing',          'slack:deploy:updated'
```
Comment (csalvato) [inline lib/slackistrano/tasks/slack.rake:45]: I am not very well versed in Slackistrano, but it looks like this totally removes the final updated message. Don't we still want that message, just later?

Comment (simon-kwame) [reply lib/slackistrano/tasks/slack.rake:45]: Yeah, I will introduce the replacement in our code.

Here:

https://github.com/LearnZillion/learnzillion/blob/01c764e0b457d45bc46892dc57b44bfd61b88d63/lib/capistrano/tasks/notify.rake#L60

Comment (simon-kwame) [reply lib/slackistrano/tasks/slack.rake:45]: Basically, i allows us to re-introduce in LearnZillion code anyway we choose.

Comment (csalvato) [reply lib/slackistrano/tasks/slack.rake:45]: Ah I see. Thank you!

**Pull request merged**